### ISSUE TITLE
C3 v2.6.0 Generates Error

### ISSUE THREAD

**Issue by JRawly** [Codeception/c3#63]
`PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Codeception\c3\Installer::copyC3V2() must be an instance of Composer\Script\Event, null given, called in /vendor/codeception/c3/Installer.php on line 85 and defined in /vendor/codeception/c3/Installer.php:88`

The Installer.php file has been updated to use copyC3V2 which defines copyC3V2(Event $event) but doesn't allow null, line 85 calls copyC3V2 with a null variable.

**Comment by bthery** [comment]
The nightly build for our application just fail because of the same issue here.

**Comment by romanmartushev** [comment]
Having the same issue here.

**Comment by Naktibalda** [comment]
Fixed in 2.6.1

**Comment by gnumoksha** [comment]
It seems 2.6.1 is still broken

```
  - Updating codeception/c3 (2.6.0 => 2.6.1): Loading from cache

Fatal error: Uncaught TypeError: Argument 1 passed to Codeception\c3\Installer::copyC3V2() must be an instance of Composer\Script\Event, null given, called in /var/www/flux/vendor/codeception/c3/Installer.php on line 85 and defined in /var/www/flux/vendor/codeception/c3/Installer.php:88
Stack trace:
#0 /var/www/flux/vendor/codeception/c3/Installer.php(85): Codeception\c3\Installer->copyC3V2(NULL)
#1 /var/www/flux/vendor/codeception/c3/Installer.php(111): Codeception\c3\Installer->copyC3(Object(Composer\Installer\PackageEvent))
#2 [internal function]: Codeception\c3\Installer->askForUpdate(Object(Composer\Installer\PackageEvent))
#3 phar:///usr/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Composer\Installer\PackageEvent))
#4 phar:///usr/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(116): Composer\EventDispatcher\EventDispatcher->doDispatch(Object(Composer\Installer\PackageEvent))
#5 phar:///usr/bin/composer/src/Composer/Installer.php(636): Composer\EventDi in /var/www/flux/vendor/codeception/c3/Installer.php on line 88

Exited with code exit status 255
CircleCI received exit code 255
```
